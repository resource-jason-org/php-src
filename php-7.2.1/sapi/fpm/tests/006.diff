001+ sh: 1: -F: not found
002+ 
003+ Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
001- [%d-%s-%d %d:%d:%d] NOTICE: fpm is running, pid %d
002- [%d-%s-%d %d:%d:%d] NOTICE: ready to handle connections
004- IPv6 ok
005+ 
006+ Warning: fsockopen(): unable to connect to [::1]:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
007+ IPv6 error