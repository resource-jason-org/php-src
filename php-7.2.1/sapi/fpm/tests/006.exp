[%d-%s-%d %d:%d:%d] NOTICE: fpm is running, pid %d
[%d-%s-%d %d:%d:%d] NOTICE: ready to handle connections
IPv4 error
IPv6 ok