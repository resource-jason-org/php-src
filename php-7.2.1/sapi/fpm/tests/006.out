sh: 1: -F: not found

Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
IPv4 error

Warning: fsockopen(): unable to connect to [::1]:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
IPv6 error